clients/adapter-amqp/src/main/java/org/eclipse/hono/adapter/client/command/amqp/ProtonBasedCommandRouterClient.java

```diff
@@ -65,9 +65,12 @@ public ProtonBasedCommandRouterClient(
             final HonoConnection connection,
             final SendMessageSampler.Factory samplerFactory,
             final ProtocolAdapterProperties adapterConfig) {
```

Comment:
The `adapterConfig` parameter is now not used anymore.

This also applies to:
ProtonBasedDeviceConnectionClient, ProtonBasedCredentialsClient, ProtonBasedDeviceRegistrationClient, ProtonBasedTenantClient
(so overall 5 classes)

Reply:
damn, missed that ...